service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/CaseSubmissionController.java

```diff
@@ -38,17 +41,23 @@
     private final DocumentGeneratorService documentGeneratorService;
     private final UploadDocumentService uploadDocumentService;
     private final ApplicationEventPublisher applicationEventPublisher;
+    private final CaseSubmissionValidatorService caseSubmissionValidatorService;
+    private final ObjectMapper mapper;
 
     @Autowired
     public CaseSubmissionController(
         UserDetailsService userDetailsService,
         DocumentGeneratorService documentGeneratorService,
         UploadDocumentService uploadDocumentService,
+        CaseSubmissionValidatorService caseSubmissionValidatorService,
```

Comment:
Can you rename it to be just `CaseValidatorService`? In the class itself there is nothing that would make it about submission. Annotations you added to domain classes are not specific to case submission, they are case specific.